### PR TITLE
Add IFF.IN_USE feature.

### PR DESCRIPTION
**Pull request by litghost**
This addresses missing features in https://github.com/SymbiFlow/symbiflow-arch-defs/issues/1536

### CODE REVIEW

**Review by mkurc-ant — COMMENTED** [review]
One comment, otherwise LTGM.

The CI reports duplicated tag `CLK_BUFG_TOP_R.CLK_BUFG_BUFGCTRL9_I1.CLK_BUFG_IMUX29_2`. But I'm not sure how does this relate to this PR. Changes to `dbfixup.py` shouldn't cause that.

File: fuzzers/035b-iob-iserdes/top.py [hunk 1]
```diff
@@ -41,9 +40,6 @@ def gen_sites():
         # Find IOI tile adjacent to IOB
         for suffix in ["IOI3", "IOI3_TBYTESRC", "IOI3_TBYTETERM"]:
```
[the remark this answers is not in the document]
Comment (litghost) [reply fuzzers/035b-iob-iserdes/top.py]: Yep, done

**Comment by litghost** [comment]
> The CI reports duplicated tag `CLK_BUFG_TOP_R.CLK_BUFG_BUFGCTRL9_I1.CLK_BUFG_IMUX29_2`. But I'm not sure how does this relate to this PR. Changes to `dbfixup.py` shouldn't cause that.

Looking into it

**Comment by litghost** [comment]
> > The CI reports duplicated tag `CLK_BUFG_TOP_R.CLK_BUFG_BUFGCTRL9_I1.CLK_BUFG_IMUX29_2`. But I'm not sure how does this relate to this PR. Changes to `dbfixup.py` shouldn't cause that.
> 
> Looking into it

Still trying to replicate this failure, without success so far.

**Comment by litghost** [comment]
> > > The CI reports duplicated tag `CLK_BUFG_TOP_R.CLK_BUFG_BUFGCTRL9_I1.CLK_BUFG_IMUX29_2`. But I'm not sure how does this relate to this PR. Changes to `dbfixup.py` shouldn't cause that.
> > 
> > 
> > Looking into it
> 
> Still trying to replicate this failure, without success so far.

I've replicated this failure, investigating cause and fixes.

**Comment by litghost** [comment]
> > > > The CI reports duplicated tag `CLK_BUFG_TOP_R.CLK_BUFG_BUFGCTRL9_I1.CLK_BUFG_IMUX29_2`. But I'm not sure how does this relate to this PR. Changes to `dbfixup.py` shouldn't cause that.
> > > 
> > > 
> > > Looking into it
> > 
> > 
> > Still trying to replicate this failure, without success so far.
> 
> I've replicated this failure, investigating cause and fixes.

I think we have a fix.

**Pull request merged**